### DISCUSSION TITLE
Build Feature : Ugly preview

### DISCUSSION THREAD

**Discussion by Wurd88** [glucauze/sd-webui-faceswaplab#47] [Q&A]
Hi, I recently updated and went to try the feature for building a face checkpoint. I noticed that all the faces that it has been creating now have been coming out blurry. Any idea what's going on?

**Comment by glucauze** [comment]
Hi, this is normal. The resulting image represents the actual image resolution (128px) provided by insightface. Previously, I used upscaling to give a better image quality (which could be time-consuming and/or lead to errors if LDSR wasn't installed).

Checkpoints, on the other hand, should function exactly as before.

If you find this disturbing, I could try upscaling again, but it won't make the checkpoints any better. Just the preview.

**Comment by glucauze** [comment]
I have fixed it in future 1.2.2 version
